app/api/V1/views/registration.py

```diff
@@ -47,7 +50,7 @@ def post(self):
 
 
 		if new_user == 'User not found':
```

Comment:
What is the logic behind this line of code

Reply:
It checks if the user is not registered then it allows the user to sign up.